cloud

```diff
@@ -2199,6 +2214,7 @@ function execute() {
     declare smtp_mocked="false"
     declare test="false"
     declare log_level="INFO"
+    declare optimize="Zero"
```

Comment:
This would have the opposite effect of what you're trying to do. Python doesn't understand spelled-out numbers, and would simply interpret this value as "non-empty" string, which would be equivalent to `PYTHONOPTIMIZE=1`. See the docs I link above.

Reply:
Okay sir